tokio/src/sync/mpsc/bounded.rs

```diff
@@ -367,6 +367,22 @@ impl<T> Receiver<T> {
         self.chan.close();
     }
 
+    /// Checks if the channel has been closed. This happens when the
+    /// [`Receiver::close`] method is called.
+    ///
+    /// [`Receiver::close`]: crate::sync::mpsc::Receiver::close
```

Comment:
The channel also closes automatically if the `Receiver` is dropped. This documentation should probably mention that as well.

Reply:
Yup I saw the doc for `is_closed` on `Tx` mention this but thought this wouldn't make sense on the Rx because we can only have a single Rx and if we drop it, we would never be able to check for `is_closed` on it. LMK if that is a terrible way to interpret things and if I should still go ahead and add the bit about being dropped.